server/prisma/schema.prisma

```diff
@@ -218,6 +218,7 @@ model refunds {
   id            Int           @id @default(autoincrement())
   orderid_fk    Int
   refund_intent String        @unique @db.VarChar(255)
+  refund_status state         @default(in_progress)
```

Comment:
It looks like default value in the schema does not align with the migration.

Reply:
in sync now